packages/dataviews/src/types.ts

```diff
@@ -124,6 +134,12 @@ export type NormalizedField< Item > = Field< Item > & {
 	label: string;
 	getValue: ( args: { item: Item } ) => any;
 	render: ComponentType< { item: Item } >;
+	sort: (
+		a: Item,
+		b: Item,
+		direction: SortDirection,
+		getValue: ( args: { item: Item } ) => any
```

Comment:
I think we should remove `getValue` from here. It's redundant with the field itself, we're defining the field here. Instead we should update the "normalize" function to inject it when we fallback to the field type sort function.